pipelines/APTemplate.yaml

```diff
@@ -0,0 +1,67 @@
+description: The AP template building pipeline specialized for DECam
```

Comment:
Period at end of sentence?

Is this designed for a specific dataset? If so, would be worth mentioning that here.

Reply:
It's not designed for a specific dataset, it should work for anything DECam. It should be easy to adapt for other cameras, some of the configs (like isr.doDark!) may need to change.